php-checks/src/main/java/org/sonar/php/checks/ParameterSequenceCheck.java

```diff
@@ -48,24 +51,47 @@
   public void visitFunctionCall(FunctionCallTree tree) {
     // only check method and function calls expect constructors with more than 1 argument
     if (!tree.getParent().is(Kind.NEW_EXPRESSION) && tree.arguments().size() > 1) {
-      Symbol symbol = getDeclarationSymbol(tree);
-      if (symbol != null && symbol.declaration() != null && symbol.is(Symbol.Kind.FUNCTION)) {
-        checkParameterSequence(tree, (NameIdentifierTree) symbol.declaration());
+      if (tree.callee().is(Kind.NAMESPACE_NAME)) {
+        checkFunctionCall(tree);
+      } else {
+        checkMethodCall(tree);
       }
     }
 
     super.visitFunctionCall(tree);
   }
 
+  private void checkMethodCall(FunctionCallTree tree) {
+    Symbol symbol = getDeclarationSymbol(tree);
```

Comment:
Ideally, we should add a new method on `FunctionCallTreeImpl` to get a `FunctionSymbol`. That will require some work on the symbol table side: later.